### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AlexandraGres**
This is my first pull request :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1377,3 +1377,6 @@ I am sure it will be an awesome journey!
 # Hello there!
 1st time working via command line, [Vasiliy](https://github.com/Vas-P) =)
 here's a potato ![Potato](https://blog.printsome.com/wp-content/uploads/potato-printing.jpg)
+
+# Hi guys!
+nice to join such a wonderfull jorney! ![Cat] (https://www.google.com/url?sa=i&url=https%3A%2F%2Fgiphy.com%2Fgifs%2Fcomputer-cat-wearing-glasses-VbnUQpnihPSIgIXuZv&psig=AOvVaw1qjuR2HA8jkhVMxiUD9ZiX&ust=1602664350071000&source=images&cd=vfe&ved=0CAIQjRxqFwoTCOiu1o2UsewCFQAAAAAdAAAAABAD)
```
Comment (nazmariam) [inline README.md:1382]: Hi, there is a url with a lot of parameters which redirects user to `giphy.com`, thats why it couldn't be displayed as a markdown preview image. Please find the source path of an image and use it

**Review by nazmariam — CHANGES_REQUESTED** [review]
Good job! But please fix image url first

File: README.md (hunk 1 above)
Comment (Temu4) [inline README.md:1382]: As well as exrta space in the embedded image construction

Comment (AlexandraGres) [reply README.md]: fixed, thank you!

Comment (AlexandraGres) [reply README.md]: fixed, thank you!

File: README.md [hunk 2]
```diff
@@ -1378,9 +1378,14 @@ I am sure it will be an awesome journey!
 1st time working via command line, [Vasiliy](https://github.com/Vas-P) =)
 here's a potato ![Potato](https://blog.printsome.com/wp-content/uploads/potato-printing.jpg)
 
+
+# Hi guys!
+nice to join such a wonderfull jorney! ![Cat](https://gph.is/g/4o6B0Km)
```
Comment (Temu4) [inline README.md:1383]: Need more changes...
According to [markdown images syntax](https://www.markdownguide.org/basic-syntax/#images) if you want to add not just link, but image preview you should use **path to file source**.

<details>
  <summary><b>Warning:</b> click only if you can't handle by your self!</summary>
<code>funny cat</code>
</details>

**Review by Temu4 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Temu4) [reply README.md]: Looks good! Try to be more attentive 👀

**Review by Temu4 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1378,13 +1378,19 @@ I am sure it will be an awesome journey!
...
+# Hi guys!
+nice to join such a wonderfull jorney! ![funny cat](https://i.giphy.com/media/VbnUQpnihPSIgIXuZv/giphy.webp)
+
 ---
 # Hello, Kottans! 
 
 I`m *[Hanna](https://github.com/HannaSyn)*, and I am really excited to join community! :smiling_face_with_three_hearts:
 ![kitten](https://slm-assets.secondlife.com/assets/16934237/view_large/animals_05.jpg) 
 ---
+
```
Comment (nazmariam) [inline README.md:1391]: Nice job! But I think you should avoid such changes, please remove blank lines

**Comment by nazmariam** [comment]
Just a little more: please resolve conflicts and we'll done

**Review by nazmariam — APPROVED** [review]

**Pull request merged**